stdlib/2and3/audioop.pyi

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Optional, Tuple
+
+AdpcmState = Tuple[int, int]
+RatecvState = Tuple[int, Tuple[Tuple[int, int], ...]]
+
+class error(Exception): ...
+
+def add(fragment1: bytes, fragment2: bytes, width: int) -> bytes: ...
+def adpcm2lin(adpcmfragment: bytes, width: int, state: Optional[AdpcmState]) -> Tuple[bytes, AdpcmState]: ...
+def alaw2lin(fragment: bytes, width: int) -> bytes: ...
+def avg(fragment: bytes, width: int) -> int: ...
+def avgpp(fragment: bytes, width: int) -> int: ...
+def bias(fragment: bytes, width: int, bias: int) -> bytes: ...
+def byteswap(fragment: bytes, width: int) -> bytes: ...
+def cross(fragment: bytes, width: int) -> int: ...
+def findfactor(fragment: bytes, reference: bytes) -> float: ...
+def findfit(fragment: bytes, reference: bytes) -> Tuple[int, float]: ...
+def findmax(fragment: bytes, length: int) -> int: ...
+def getsample(fragment: bytes, width: int, index: int) -> int: ...
+def lin2adpcm(fragment: bytes, width: int, state: Optional[AdpcmState]) -> Tuple[bytes, AdpcmState]: ...
+def lin2alaw(fragment: bytes, width: int) -> bytes: ...
+def lin2lin(fragment: bytes, width: int, newwidth: int) -> bytes: ...
+def lin2ulaw(fragment: bytes, width: int) -> bytes: ...
+def max(fragment: bytes, width: int) -> int: ...
+def maxpp(fragment: bytes, width: int) -> int: ...
+def minmax(fragment: bytes, width: int) -> Tuple[int, int]: ...
+def mul(fragment: bytes, width: int, factor: float) -> bytes: ...
+def ratecv(
+    fragment: bytes,
+    width: int,
+    nchannels: int,
+    inrate: int,
+    outrate: int,
+    state: Optional[RatecvState],
+    weightA: Optional[int] = ...,
```

Comment:
It doesn't look like these are really `Optional`; they default to integer values.

Reply:
Thanks for catching this, it's now fixed.